index.d.ts

```diff
@@ -0,0 +1,71 @@
+import AccountKeyFail from './packages/caver-account/src/accountKey/accountKeyFail'
+import Account, { IAccount } from './packages/caver-account/src/index'
+import KeyringContainer from './packages/caver-wallet'
+import { ITransaction } from './packages/caver-transaction'
+import { IUtils } from './packages/caver-utils'
+import RPC from './packages/caver-rpc'
+import KCT from './packages/caver-kct'
+import { IABI } from './packages/caver-abi'
+import { IContract } from './packages/caver-contract'
+import IPFS from './packages/caver-ipfs'
+
+export class Caver {
+    /**
+     * @deprecated it is not official property
+     */
+    static utils: any
+    /**
+     * @deprecated it is not official property
+     */
+    static providers: any
+    /**
+     * @deprecated it is not official property
+     */
+    static abi: any
+
+    constructor(provider: any, net?: any)
+
+    account: IAccount
```

Comment:
Why are some types defined as interfaces? By what criteria is it divided?

Reply:
@jimni1222 
the reason is same with above